### DISCUSSION TITLE
shortcut tied to unknown plugging and breaking other shortcuts

### DISCUSSION THREAD

**Discussion by jin-ahn** [LazyVim/LazyVim#4652] [Q&A] [answered]
Recently updated lazyvim after not touching it for a while.

I have a bunch of keymaps for the "mini" plugins such as mini.commment, that start with "m". An example is in N and V "mc" will comment or "mcc" will comment a line.

There is a new plugin or feature or keymap that is completely breaking this ability. Now when I press "mc" it will add a mark in the gutter with "c". In fact anytime I press m+any key it will mark the gutter with that key. I have no idea what this plug in is. I have searched for keymaps and nothing shows. It's driving me crazy. Anyone know how I can fix this?
image.

I have double checked I don't have anything configured in my keymaps for jsut "m" and I can't tell what plugin is doing this. I dont have  marks.nvim
![Screenshot_2024-10-25 01 19 54_U4VoNT](https://github.com/user-attachments/assets/651467c1-0e25-47b4-804f-832bf1202368)

**Comment by dpetka2001** [comment, marked as answer]
That is Neovim default behavior for setting marks, not a plugin (and it's been there for ages) . See `:h m` for info.
